minesweeper.py

```diff
@@ -1 +1,25 @@
-import pygame
\ No newline at end of file
+import pygame
+
+def findSurroundingBombs(arr, row, col):
```

Comment:
```suggestion
def find_surrounding_bombs(arr, row, col):
```

ik you are pretty used to writing Java code, but with Python, functions and variables should not use camelCase, but rather snake_case ([stated in PEP8](https://peps.python.org/pep-0008/#function-and-variable-names)). It is weird at first to make that adjustment, but over time, adjusting between the different styles isn't an issue.